### PR TITLE
hysteria: Add version 1.0.1

### PR DESCRIPTION
**Pull request by Fei1Yang**
Closes #XXXX

Relates to #XXXX

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Append `#/hysteria.exe` at the end of all URLs, and change the bin entry to `hysteria.exe`

**Comment by Fei1Yang** [comment]
Both issues are fixed.

**Comment by rashil2000** [comment]
If this is a CLI-only app, can you transfer this PR to the Main bucket?

**Comment by Fei1Yang** [comment]
I don't think this app fits criteria "a reasonably well-known and widely used developer tool" due to a relatively small userbase, can you confirm that you want this app to be added to the Main bucket?

**Comment by rashil2000** [comment]
> I don't think this app fits criteria "a reasonably well-known and widely used developer tool" due to a relatively small userbase

Okay, then it's fine to add it here

File: bucket/hysteria.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.0.1",
+    "description": "Hysteria is a feature-packed network utility optimized for networks of poor quality (e.g. satellite connections, congested public Wi-Fi, connecting from China to servers abroad)",
```
Comment (rashil2000) [inline bucket/hysteria.json:3]:
```suggestion
    "description": "Network utility optimized for networks of poor quality (e.g. satellite connections, congested public Wi-Fi, connecting from China to servers abroad)",
```

Comment (Fei1Yang) [reply bucket/hysteria.json]: Description updated.

**Pull request merged**